src/components/Dao/DaoHeader.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import { IDAOState, Scheme } from "@daostack/client";
+
+import withSubscription, { ISubscriptionProps } from "components/Shared/withSubscription";
+import AccountImage from "components/Account/AccountImage";
+import * as css from "./DaoHeader.scss";
+
+const styles = { 
+  circularSquare: {
+    borderRadius: "50%",
+    borderColor: "white",
+    borderStyle: "solid",  
+  },
+};
+
+interface Signal {
+  id: string;
+  data: any | string;
+}
+
+interface IExternalProps {
+  daoState: IDAOState;
+  signal?: Signal | any;
+}
+
+type IProps = IExternalProps & ISubscriptionProps<[Scheme[], Signal | any]>;
+
+class DaoHeaderComponent extends React.Component<IProps, any> {
+  render() {
+    const { daoState, signal } = this.props;
+    const { name, memberCount, address } = daoState;
+    const data = {
+      daoImg: "/assets/images/generic-user-avatar.png",
+      reputationHolders: memberCount,
+      holdings: [
+        { name: "GEN", amount: 0.24653 },
```

Comment:
where are these numbers coming from?

Reply:
Sorry super late response. I had to take time off due to personal issues. I am back now.

What I am currently working on is on making it responsive and render "dao members count" and "dao holdings".

Another todo is when we have a signal scheme proposal we want to be able to show in the proposal summary what off chain data are we modifying so that users know what to expect to change if a signal scheme proposal is passed.